cli/cmd/lql_sources.go

```diff
@@ -83,6 +83,7 @@ func listQuerySources(_ *cobra.Command, args []string) error {
 			getListQuerySourcesTable(datasourcesResponse.Data),
 		),
 	)
+	cli.OutputHuman("Use 'lacework query show-source <datasource_id>' to show details about the data source.\n")
```

Comment:
Can we add a new line in between the data shown and the breadcrumb?
```suggestion
	cli.OutputHuman("\nUse 'lacework query show-source <datasource_id>' to show details about the data source.\n")
```